src/ResourceManager/RecoveryServices.Backup/ChangeLog.md

```diff
@@ -1,83 +0,0 @@
-<!--
-    Please leave this section at the top of the change log.
```

Comment:
Please move the two "Changelog.md" files into the respective Commands.RecoveryServices.Backup and Commands.RecoveryServices folders.  This will allow us to do versioning on the two modules separately.  Same comment for the documentation folder that contains breaking changes.